graphql-dynamodb-manager-test/pom.xml

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.fleetpin</groupId>
+        <artifactId>graphql-dynamodb-manager</artifactId>
+        <version>0.1.3</version>
+    </parent>
+
+    <artifactId>graphql-dynamodb-manager-test</artifactId>
+
+    <properties>
+        <junit.jupiter.version>5.6.0</junit.jupiter.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>sonatype</id>
+            <name>central snapshot</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>sonatype</id>
+            <name>central release</name>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+        </repository>
+    </distributionManagement>
+
+    <scm>
+        <url>https://github.com/fleetpin/graphql-dynamodb-manager</url>
+        <connection>scm:git:https://github.com/fleetpin/graphql-dynamodb-manager.git</connection>
+        <developerConnection>scm:git:https://github.com/fleetpin/graphql-dynamodb-manager.git</developerConnection>
+    </scm>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Ashley Taylor</name>
+            <email>ashley.taylor@fleetpin.co.nz</email>
+            <organization>Fleetpin</organization>
+            <organizationUrl>http://www.fleetpin.co.nz</organizationUrl>
+        </developer>
+        <developer>
+            <name>Alexander Johnston</name>
+            <email>alexander.johnston@fleetpin.co.nz</email>
+            <organization>Fleetpin</organization>
+            <organizationUrl>http://www.fleetpin.co.nz</organizationUrl>
+        </developer>
+    </developers>
+
+    <dependencies>
+        <dependency>
```

Comment:
will pull dynamo and aws dependencies from other modules

Reply:
Removed.